lib/libipmeta.h

```diff
@@ -310,6 +339,13 @@ const char *ipmeta_get_provider_name(ipmeta_provider_t *provider);
  */
 ipmeta_provider_t **ipmeta_get_all_providers(ipmeta_t *ipmeta);
 
+/* Destructor that frees every column of record whenever record is not NULL*/
```

Comment:
Please add doxygen-style documentation like on other public functions.
(i.e., add documentation for parameters and return values)